macrosynergy/visuals/README.md

```diff
@@ -0,0 +1,186 @@
+# macrosynergy.visuals
+
+## Description
+
+The `macrosynergy.visuals` subpackage contains functions for visualizing quantamental data, and to provide `view.*` functions for the `macrosynergy` package.
+
+The functionality is built around the ability to create quick generic plots of data, and to provide a framework for creating custom plots.
+Simplicity, speed, consistency and flexibility are the main goals.
+The core visualization functions are built around `matplotlib`, with goals of mimicking the rich and dataframe-friendly nature of `seaborn`.
```

Comment:
The core visualization functions are built around `matplotlib`, with goals of mimicking the rich and dataframe-friendly nature of `seaborn` without the overheads it comes with.